backend/src/routes/portfolioRoutes.ts

```diff
@@ -0,0 +1,16 @@
+import express from 'express';
+
+import {createPortfolio, deletePortfolio, getPortfolios, modifyFunds, updatePortfolio} from '../controllers/portfolioController';
+import {authHandler} from '../middleware/authHandler';
+
+export const portfolioRouter = express.Router();
+
+portfolioRouter.post('/', authHandler, createPortfolio);
+
+portfolioRouter.get('/:id', authHandler, getPortfolios);
+portfolioRouter.get('/', authHandler, getPortfolios);
+
+portfolioRouter.put('/:id', authHandler, updatePortfolio);
+portfolioRouter.put('/addFund/:id', authHandler, modifyFunds);
```

Comment:
```suggestion
portfolioRouter.put('/modifyFund/:id', authHandler, modifyFunds);
```
Bc funds can be added/removed so I think this is better name.